internal/client.go

```diff
@@ -794,6 +798,13 @@ type (
 		onConflictOptions *OnConflictOptions
 	}
 
+	// startWorkflowResultInfo can be passed to StartWorkflowOptions to receive additional information
+	// of StartWorkflowExecution response.
+	startWorkflowResultInfo struct {
```

Comment:
Can we change this to "ResponseInfo" instead of "ResultInfo" so it's not confused with the concept of "workflow result"?